src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,16 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/cinema_app?serverTimezone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">bna123456</property>
+        <property name="show_sql">true</property>
+        <property name="hibernate.hbm2ddl.auto">create-drop</property>
```

Comment:
if you apply update option, you'll be able to test your try-catch construction in main. Now your construction (try-catch in maim, I mean) has no matter, because you don't test option if you add user with email which exists in DB.